src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -145,6 +145,28 @@
           class="viewCount"
         > • {{ $tc('Global.Counts.Watching Count', viewCount, {count: parsedViewCount}) }}</span>
       </div>
+      <div
+        v-if="is4k || hasCaptions"
+        class="videoTagLine"
+      >
+        <div
+          v-if="is4k"
+          class="videoTag"
+          :aria-label="$t('Search Listing.Label.4K')"
```

Comment:
I actually based these values off the Youtube website. I believe they used role "img" because it behaves like a image label. It's probably because they want it behave like this - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Img_role#using_roleimg_to_confer_meaning_that_is_obscured_or_implied

My guess about usage of the aria-label on a screen reader is that it will prefer to read the aria-label over the actual text inside. The label on the captions is more descriptive than the text so it made sense to me. 4k label I guess didn't need more elaboration.

Reply:
Thanks, that makes sense.